RickyChien.md

```diff
@@ -1,13 +1,10 @@
-##  ~ 08/05
+##  ~ 08/12
 
 [Control Center]
 
-* [r+] [Bug 1291998](http://bugzil.la/1291998) - Background color of cancel button in permission dropdown on Windows is distinct from OS X and Linux
- * Styling issue of cancel button on Windows. Already got r+ and ready to land. 
+* [Fixed] [Bug 1291998](http://bugzil.la/1291998) - Background color of cancel button in permission dropdown on Windows is distinct from OS X and Linux
+* [r?] [Bug 1206229](http://bugzil.la/1206229) - Inform the user that changes may require a page reload
 
 [DevTools]
 
-* [WIP] [Bug 1278923](http://bugzil.la/1278923) - Exported HAR format is missing Content-Type and Content-Length headers
-* [Fixed] [Bug 1286283](http://bugzil.la/1286283) - HTML ToolSidebar should support ARIA
-* [Fixed] [Bug 1102269](http://bugzil.la/1102269) - [highlighter] infobar can be outside visible area
-* [Fixed] [Bug 1291618](http://bugzil.la/1291618) - Fix dropmarker position of command-button-frames in RTL locales
+* [r?] [Bug 1278923](http://bugzil.la/1278923) - Exported HAR format is missing Content-Type and Content-Length headers
```

Comment:
Can you move `[r+]` etc. to the next line next week? It would help me understand the diff output, thank you!